pom.xml

```diff
@@ -925,6 +926,11 @@
         <version>58.2</version>
       </dependency>
       <dependency>
+        <groupId>com.rometools</groupId>
+        <artifactId>rome</artifactId>
+        <version>1.5.1</version>
```

Comment:
rome version 1.5.1 is to old and uses JDOM(1).
(https://github.com/rometools/rome/blob/v1.5.1/pom.xml)
You should switch to at least 1.6.0 which is using jdom2